package.json

```diff
@@ -679,6 +679,28 @@
                 "when": "explorerViewletVisible && filesExplorerFocus",
                 "intellij": "Rename (File)"
             },
+            {
+                "key": "ctrl+alt+m",
+                "mac": "cmd+alt+m",
+                "command": "editor.action.codeAction",
```

Comment:
Cool!
[Keybindings for Code Actions](https://code.visualstudio.com/docs/editor/refactoring#_keybindings-for-code-actions)